teamscale-client/src/main/java/com/teamscale/client/ITeamscaleService.java

```diff
@@ -27,8 +27,9 @@
 			@Path("projectName") String projectName,
 			@Query("format") String format,
 			@Query("t") CommitDescriptor commit,
-			@Query("adjusttimestamp") boolean adjustTimestamp,
-			@Query("movetolastcommit") boolean moveToLastCommit,
+			@Query("revision") String revision, 
+			@Query("adjusttimestamp") Boolean adjustTimestamp,
```

Comment:
Sorry, I know this was already here, but the `adjusttimestamp` has already been removed in TS 2.x. Could you remove it?

Reply:
Good point.